canonical/ubuntu.com
# K8s docs various updates and link fixes

## Done

-Update metallb instructions
- update cilium
- add note for LXD install re Calico
- Openstack --> OpenStack in various places
- remove stray <woke-ignore> tags which interfere with rendering
- update all links which used to point to `/juju.is/docs/olm/*` to point to `/juju.is/docs/juju/*`
- some other link updates

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
    - Be sure to test on mobile, tablet and desktop screen sizes
- N.B. For QA purposes, note that the text and content of changes have already been reviewed
